content/05.discussion_and_conclusion.md

```diff
@@ -0,0 +1,27 @@
+## Discussion and Conclusions
+
+Filling out knowledge bases via manual curation can be an arduous and erroneous task [@doi:10.1093/bioinformatics/btm229].
+Using manual curation alone becomes impractical as the rate of publications continuously increases.
+Data programming is a paradigm that uses label functions to speed up the annotation process and can be used to solve this problem.
+However, creating useful label functions is an obstacle to this paradigm, which takes considerable time.
+We tested the feasibility of re-using label functions to reduce the number of label functions required for strong prediction performance.
+
+Through our sampling experiment, we found that adding edge-specific label functions increases performance for the generative model (Figure {@fig:auroc_gen_model_test_set}).
```

Comment:
```suggestion
Our sampling experiment revealed that adding edge-specific label functions increases performance for the generative model.
```
If one is citing figures here, it's probably too detailed. I recommend citing figures almost exclusively within the results.